collections/retail/general-notifications/templates/phone-number-deleted-email/template.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name":"phone-number-deleted-email",
+  "description":"Email template for phone number deleted notification",
+  "title":"Phone number deleted",
+  "engagement": "general-notification",
+  "channel": "email",
+  "placeholderGroups":["phoneNumberDeleted"]
+}
```

Comment:
@AnjBB72 I can see you are using placeholders from user profile on the template as well, so here you should update the last line:
 "placeholderGroups":["phoneNumberDeleted", "userProfile"]

Reply:
hey @AnjBB72 is userProfile not user-profile